README.md

```diff
@@ -1,10 +1,88 @@
+However, a user may wish to install the carta-frontend component separately in order to test beta versions, for example, or to develop it.
+
+## Pre-built production packages
+The simplest way to install the carta-frontend component is to use the pre-built production packages from NPM (any OS with Node.js installed), our PPA (Debian packages for Ubuntu), our RPM repository (for RHEL based systems), or Homebrew (for MacOS).
+
+### NPM
+
+Pre-built production carta-frontends can be obtained from the [npmjs.com carta-frontend repository](https://www.npmjs.com/package/carta-frontend?activeTab=explore).
+
+If you have Node.js and `npm` already installed on your system, a production carta-frontend can be installed using the following command:
+
+```
+npm i carta-frontend
+```
+The files will be installed to your `node-modules` directory.
+Specific carta-frontend versions can be installed by appending `@` and the version number. For example,
+```
+npm i carta-frontend@1.4.0
+npm i carta-frontend@2.0.0-beta.0
+```
+The available versions are listed in the [npmjs.com carta-frontend versions tab](https://www.npmjs.com/package/carta-frontend?activeTab=versions).
+
+Alternatively, the pre-built production carta-frontend can be downloaded directly without npm. For example,
+```
+wget https://registry.npmjs.org/carta-frontend/-/carta-frontend-2.0.0.tgz
+```
+The `tgz` extracts to `package/build`. 
+
+The carta-backend can be started up to use any non-default carta-frontend by specifying its location with the `--frontend_folder` flag. For example,
+```
+./carta-backend --frontend_folder $HOME/package/build
+```
+
+### PPA
+
+A Personal Package Archive (PPA) is a third-party package repository for Ubuntu. We supply pre-built production carta-frontends at the [CARTAvis-team PPA on the Ubuntu Lauchpad](https://launchpad.net/~cartavis-team/+archive/ubuntu/carta).
+
+To install the latest release version:
+```
+sudo add-apt-repository ppa:cartavis-team/carta
+sudo apt-get update
+sudo apt-get install carta-frontend
+```
+
+The default install location for the carta-frontend is `/usr/local/share/carta/frontend/`
+
+### RPM
+
+RPMs are for users of Red Hat Enterprise Linux (RHEL) and other Red Hat-based Linux distributions, i.e. CentOS. 
+To install the latest release version, add the appropriate repository file depending on whether you are using RHEL7 (**el7**) or RHEL8 (**el8**):
+
+```
+sudo curl https://packages.cartavis.org/cartavis-el7.repo --output /etc/yum.repos.d/cartavis.repo
+```
+or
+```
+sudo curl https://packages.cartavis.org/cartavis-el8.repo --output /etc/yum.repos.d/cartavis.repo
+```
+Then the carta-frontend component can be installed using `yum` or `dnf` :
+```
+sudo yum install carta-frontend
+```
+The default install location for the carta-frontend RPM is `/usr/local/share/carta/frontend/`
```

Comment:
Again, "the RPM package install location".

Is this really the right location? Surely on RedHat system packages should also install to `/usr/`, not to `/usr/local/`?